protocols/ping/src/handler.rs

```diff
@@ -261,9 +257,10 @@ impl ConnectionHandler for Handler {
                     log::debug!("Inbound ping error: {:?}", e);
                     self.inbound = None;
                 }
-                Poll::Ready(Ok(stream)) => {
+                Poll::Ready(Ok(mut stream)) => {
                     log::trace!("answered inbound ping from {}", self.peer);
 
+                    stream.ignore_for_keep_alive();
```

Comment:
Why not call `ignore_for_keep_alive` as soon as the stream is handed to the `ConnectionHandler`, i.e. within `on_connection_event`?

Reply:
Yes good call, we should do that as early as possible.

Reply:
changed this call within `on_connection_event ` 92823e1